cmd/launcher/launcher.go

```diff
@@ -276,8 +276,9 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 	}
 
 	// Add the log checkpoints to the rungroup, and run it once early, to try to get data into the logs.
+	// The checkpointer can take up to 5 seconds to run, so do this in the background.
 	checkpointer := checkups.NewCheckupLogger(logger, k)
-	checkpointer.Once(ctx)
+	go checkpointer.Once(ctx)
```

Comment:
The prior way was intentional -- it was intending to get some of the identifying information into the logs _before_ any early crashes might occur. It's fine to revisit, maybe we don't need it with all the things we've subsequently added.

Though I think this begs the question -- why does it take 5s to log, and should we add go routines downstack?

Reply:
As suspected -- it's the connectivity checkup: https://github.com/kolide/launcher/blob/main/ee/debug/checkups/connectivity.go

Reply:
I wonder if we should just async that one check. Not sure

Reply:
That seemed like maybe overly complicated to coordinate an exclusion for that one checkup for the checkpoint only, but not doctor/flare

Reply:
Thinking about it... Maybe this simple thing is good enough for now.

We probably _do_ want to keep this in the log checkpoint, it provides an interesting point-in-time view. 

I suspect that the theoretical correct state would be to make the timer log checkpoints use a different set of debuggers than the initial startup ones. And I'm not sure it's worth doing that this moment

Reply:
Agreed!